modules/tinymce/src/themes/silver/test/ts/browser/editor/core/ContentLanguageHiddenTest.ts

```diff
@@ -0,0 +1,27 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.core.ContentLanguageHiddenTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: 'language | bold italic'
+  }, [ Theme ]);
+
+  it('TINY-7570: Does not show the toolbar button if content_langs is not defined', () => {
+    const editor = hook.editor();
+    assert.throws(() => TinyUiActions.clickOnToolbar(editor, '[title="Language"]'));
```

Comment:
Thinking more on this, I think this test should really assert that the button isn't present, currently it could pass if the button is present but throws an exception when clicked

Reply:
Good point... this really should be a `UiFinder.notExists` assertion or similar instead.